Examples/Detectors/TelescopeDetector/include/ActsExamples/TelescopeDetector/TelescopeDetectorOptions.hpp

```diff
@@ -0,0 +1,54 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Units.hpp"
+#include "ActsExamples/Utilities/Options.hpp"
+
+#include <utility>
+
+#include <boost/program_options.hpp>
+
+namespace ActsExamples {
+
+namespace Options {
+
+namespace po = boost::program_options;
+
+/// The telescope geometry options
+///
+/// @tparam options_t Type of the options object (bound to boost API)
+///
+/// @param opt The provided object, where root specific options are attached
+template <typename options_t>
+void addTelescopeGeometryOptions(options_t& opt) {
```

Comment:
This does not need to be template on the option description type as there is only `ActsExamples::Options::Description`. Then the implementation can go into a `.cpp` file.

Reply:
Right. Fixed.